docs/source/load_blast.rst

```diff
@@ -1,33 +1,47 @@
 Loading Blast results
 =====================
 
-In order to load a Blast results file, both the query and the subject records must be previously stored.
+In order to load a BLAST xml result file, both the query and the subject records must be previously stored (see below). The current version was tested for loading BLAST analysis on proteins.
 
-Load Blast
+Load BLAST subject records
+---------------------------------
+In case you did a BLAST against a multispecies protein database, like NCBI's nr or Uniprot's trembl or swissprot, you need to use the following command with the flag: --organism_subject 'multispecies multispecies' in order to load all subjects.
+
+.. code-block:: bash
+
+    python manage.py load_similarity_matches --file blast_result.xml --format blast-xml
+
+* Loading this file can be faster if you increase the number of threads (--cpu).
+
+Load BLAST
 ----------
 
+If all queries and subjects are already loaded you can run the following command to load a BLAST xml result:
+
 .. code-block:: bash
 
-    python manage.py load_similarity --file blast_result.xml --format blast-xml --so_query mRNA --so_subject assembly --program blastn --programversion 2.2.31+
+    python manage.py load_similarity --file blast_result.xml --format blast-xml --so_query polypeptide --so_subject protein_match --program diamond --programversion 0.9.24 --organism_query 'Oryza sativa' --organism_subject 'multispecies multispecies'
 
 * Loading this file can be faster if you increase the number of threads (--cpu).
 
 .. code-block:: bash
 
     python manage.py load_similarity --help
 
-================   ===================================================================
---file 			   Blast XML file *
---format		   blast-xml *
---so_query         Query Sequence Ontology term. (eg. assembly, mRNA, polypeptide) *
---so_subject       Subject Sequence Ontology term. (eg. assembly, mRNA, polypeptide) *
---program          Program *
---programversion   Program version *
---name             Name
---description      Description
---algorithm        Algorithm
---cpu 			   Number of threads
-================   ===================================================================
+==================   ========================================================================================================
+--file 		                        Blast XML file *
+--format	                        blast-xml *
+--so_query           Query Sequence Ontology term. (eg. assembly, mRNA, polypeptide) *
+--so_subject         Subject Sequence Ontology term. (eg. protein_match if loading a BLAST result) *
+--organism_query     Query's organism name. eg. 'Oryza sativa'. If not loaded previously put 'multispecies multispecies'. *
```

Comment:
If the query organism is not loaded, why someone would want to load the blast result to the database. 
I think the query should no accept multispecies